templates/test/ci/cluster-template-prow-custom-vnet.yaml

```diff
@@ -47,6 +47,8 @@ spec:
       securityGroup:
         name: node-nsg
     vnet:
+      cidrBlocks:
+      - 10.0.0.0/8
```

Comment:
for this one I would omit the cidrBlock, since the idea is that we are pre-creating the vnet out of band, we don't want the CIDR to be set (it will be ignored and overwritten with the actual value of the cidr range during reconcile)